### ISSUE TITLE
Support .env / AWS config file format (aka ini) for local dev

### ISSUE THREAD

**Issue by ryanblock** [architect/architect#618]
I'd like to propose we additively add support for `.env` / AWS config file format.

We currently ask people to work locally with a `.arc-env` file that is unusable by very common tools like [dotenv](https://www.npmjs.com/package/dotenv) (7mm d/l week). 

Although it's true to the Architect format (with pragmas, keys/values separated by spaces, etc.) It's another (seemingly proprietary) thing to explain and document.

Having potentially two env formats side by side has been, for me, problematic; it's bitten me numerous times in personal Arc projects, where my local `.env` (used for testing individual files, scripts, etc.) was out of sync with my `.arc-env`, etc.

The ability to tell people to just create a `.env` file the likes of which closely resembles AWS's config env feels like it could be a much lower bar and improve interop.

Proposed format:
```
[testing]
foo=bar
bar=baz
baz=buz

[staging]
foo=bar
bar=baz
# baz=buz commented out
```

Reference:
- [AWS's parser for this format](https://github.com/aws/aws-sdk-js/blob/master/lib/util.js#L214-L233)
- #250

**Comment by Stegosource** [comment]
Big +1 for this. I'm currently needing to keep a `.arc-env` and `.env` file in my root.

**Comment by ryanblock** [comment]
I'm on the job! Hoping this week for at least `.arcrc` (or whatever).

**Comment by wesbos** [comment]
+1 for this, would be nice to just use a .env file locally

**Comment by ryanblock** [comment]
Lol at my "next week!" comment from 7mo ago. It's in an RC tho, stand by.

**Comment by ryanblock** [comment]
Shipped in 8.3!
